api/app/src/routes/webhook/fitbit.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import { asyncHandler } from "../util";
+import { processData } from "../../command/webhook/fitbit";
+
+const routes = Router();
+
+// This value is provided on the dev.fitbit dashboard when you update or craete a new webhook subscriber.
+const FITBIT_VERIFICATION_CODE = "";
+
+/** ---------------------------------------------------------------------------
+ * GET /webhook/fitbit
+ *
+ * Verify Fitbit WH subscriber. Receives a verification notification and sends back a verification post request.
+ *
+ */
+routes.get(
+  "/",
+  asyncHandler(async (req: Request, res: Response) => {
+    if (req.query.verify) {
+      console.log(
+        "Verifying the subscriber. Make sure to use the verification code provided in the dev.fitbit dashboard!"
+      );
+      if (req.query.verify === FITBIT_VERIFICATION_CODE) {
+        console.log("Received correct verification code.");
+        return res.sendStatus(204);
+      } else {
+        console.log("Incorrect verification code detected!");
+        return res.sendStatus(404);
+      }
+    } else {
+      return res.sendStatus(200);
+    }
+  })
+);
```

Comment:
not understanding this - `FITBIT_VERIFICATION_CODE` is always an empty string?

is this a leftover? thought we were doing auth in the lambda

Reply:
When you're adding subscribers (URLs for webhooks on the [dev.fitbit.com](https://dev.fitbit.com/apps) - go into an app and see the subscribers on the bottom), you need to [manually verify](https://dev.fitbit.com/build/reference/web-api/subscription/create-subscription/#:~:text=userRevokedAccess%20collection%20does%20not%20require%20any%20scopes) them with the code that is provided on the dashboard. We'll just have to insert that string into this const here and run verification - this will only be done once per subscriber.

Reply:
See [Raf's comment](https://github.com/metriport/metriport/pull/651/files#r1258949012), we'll want to make this an env var as this just needs to be verified once per registered app subscriber URL.

Pretty much if we're hardcoding strings that need to be configured later, this is always a good candidate for an env var